app/models/user.js

```diff
@@ -23,5 +26,19 @@ export default Addressable.extend({
   organisations: hasMany('organisation', {async: false}),
   organisationsUsers: hasMany('organisationsUsers', {async: false}),
 
-  // image:       belongsTo('image', { async: false }),
+  position: Ember.computed('organisationsUsers.[]', function() {
+    const organisationsUsers = this.get("organisationsUsers");
+    return organisationsUsers.length ? 
+    organisationsUsers.get("firstObject.position") : '';
+  }),
+
+  onlineStatus: Ember.computed('lastConnected', 'lastDisconnected', function() {
```

Comment:
Maybe the returned value should be coming from an enum 
e.g 
```javascript
const CONN_STATUSES = {
    ONLINE: 'online',
    OFFLINE: 'offline',
    UNKNOWN: 'unknown'
}
```
and then can be interpolated to build the translation string `this.get('i18n').t('connection.status.${status}')`

the mix of boolean and strings is a bit weird :D

Reply:
Good idea.

Reply:
Will still need the conditions for showing online/not-connected status. So I think it'll the same thing. (Also instead of offline we show `not connected`).
I did added translations though

Reply:
Conditions are fine, it's just that the return value is very hard to utilise anywhere else if needed :)

Reply:
@Darshan-Chauhan maybe we can start a practice to name the properties that are translations with a `Label` suffix ?

e.g `onlineStatusLabel` 

That way whenever we see a name similar to this, we know that it's a translation string.
To avoid anyone doing something like `if (user.get('onlineStatus') === 'Not connected)`  (which is dangerous)
Just an idea

Reply:
I liked the idea 👍